test/system/070-build.bats

```diff
@@ -194,8 +184,14 @@ EOF
                build_test
     is "${lines[0]}" "$workdir" "container default command: pwd"
     is "${lines[1]}" "$s_echo"  "container default command: output from echo"
+
     is "${lines[2]}" "$s_env1"  "container default command: env1"
-    is "${lines[3]}" "$s_env2"  "container default command: env2"
+
+    # remote client environment is not forwarded to host
+    if ! is_remote; then
```

Comment:
I think it would be useful to make this an if-else, so we actually confirm that podman-remote doesn't somehow manage to start sending envariables:
```bash
if is_remote; then
    is "${lines[3]}" "this-should-be-overridden-by-env-host" "podman-remote does not send local environment"
else
    ...
```